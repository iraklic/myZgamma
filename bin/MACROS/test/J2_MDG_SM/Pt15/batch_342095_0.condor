000 (342095.000.000) 11/06 14:16:33 Job submitted from host: <131.225.191.129:50550>
...
001 (342095.000.000) 11/06 14:17:22 Job executing on host: <131.225.191.138:33331>
...
006 (342095.000.000) 11/06 14:17:30 Image size of job updated: 81868
...
005 (342095.000.000) 11/06 16:58:20 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 02:40:38, Sys 0 00:00:04  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 02:40:38, Sys 0 00:00:04  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	53262  -  Run Bytes Sent By Job
	3008743  -  Run Bytes Received By Job
	53262  -  Total Bytes Sent By Job
	3008743  -  Total Bytes Received By Job
...
