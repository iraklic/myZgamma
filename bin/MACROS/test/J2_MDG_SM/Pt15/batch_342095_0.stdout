
CMSSW on Condor

started at Sun Nov  6 14:17:23 CST 2011

parameter set:
CONDOR_CLUSTER: 342095
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_342095_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_342095_0.log 2>&1\n

finished at Sun Nov  6 16:58:20 CST 2011
